src/app/Shopping/Cart/Map.tsx

```
@@ -0,0 +1,81 @@
+import styled from "styled-components/native";
+import { color, Font } from "../../../styles"
+import Search from "../../../components/Main/Search";
+import { AuthButton } from "../../../components";
+import StoreLocation from "../../../components/Shopping/StoreLocation";
+import { useState } from "react";
+
+const items = ["안녕", "히히", "예에", "안녕", "히히", "예에", "안녕", "히히"];
+
+const Map = () => {
+  const [isSelect, setIsSelect] = useState<boolean>(false);
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)